rel/relationship_test.go

```diff
@@ -42,3 +43,58 @@ func ExampleRelationship_WithCaveat() {
 	// Output:
 	// document:example#viewer@user:jzelinskie[only_on_tuesday:{"day_of_the_week":"wednesday"}]
 }
+
+func ExampleRelationship_WithExpiration() {
```

Comment:
what about a quick example with caveats + expiration? not sure how it will look

Reply:
Example_ only works with named methods; I could add one but it couldn't be named `Example`